commit a781558d055c4840b2b322551e765e22d6be30f4
Author: Kai Uwe Broulik <kde@privat.broulik.de>
Date:   Tue Jun 4 00:29:12 2013 +0200

    Don't access QDBusPendingReply value if it is not valid.
    
    REVIEW: 110746
    BUG: 315017
    FIXED-IN: 4.10.5

diff --git a/powerdevil/daemon/powerdevilpolicyagent.cpp b/powerdevil/daemon/powerdevilpolicyagent.cpp
index e9d7ab3..d217e92 100644
--- a/powerdevil/daemon/powerdevilpolicyagent.cpp
+++ b/powerdevil/daemon/powerdevilpolicyagent.cpp
@@ -383,7 +383,7 @@ PolicyAgent::RequiredPolicies PolicyAgent::requirePolicyCheck(PolicyAgent::Requi
         QDBusPendingReply< bool > rp = m_ckSessionInterface.data()->asyncCall("IsActive");
         rp.waitForFinished();
 
-        if (!rp.value() && !m_wasLastActiveSession) {
+        if (!(rp.isValid() && rp.value()) && !m_wasLastActiveSession) {
             return policies;
         }
     }
